iset.mm

```diff
@@ -146440,6 +146440,778 @@ since the target of the homomorphism (operator ` O ` in our model) need
+      TKZXFWMUUGKZWOWSNOZFBOZWOUUCBOZXCXEUUHXJWPWTWKUUJUUKPWHXJWLUUGXKQWMWPWTUL
+      WMWPWTUMWMWKUUGYBRZABCWOWSFGHUNUPWMUUJXCPUUGWMUUIXBFBWMDEXTYAWBUSRUUHUUKD
+      UUCYCSZBOZXEUUHWHWIUUCAIZUUKUUNPWMWHUUGYIRZWMWIUUGXGRUUHWHWSJIZWKUUOUUPWT
+      UUQWMWPWSWCWDZUULABCWSFGHVJTABCYCDUUCGHYKVSTUUHUUMXDDBUUHWSMZFBOZUUMXDUUH
+      WHUUQWKUUTUUMPUUPUURUULABCYCWSFGHYKVHTUUHUUSEFBWMUUSEPUUGWMEYAWFRUSVAVRVM
+      WAVEWEWG $.
+
+    $( Reversed product of group multiples.  (Contributed by Paul Chapman,
+       17-Apr-2009.)  (Revised by AV, 30-Aug-2021.) $)
+    mulgassr $p |- ( ( G e. Grp /\ ( M e. ZZ /\ N e. ZZ /\ X e. B ) ) ->
+                              ( ( N x. M ) .x. X ) = ( M .x. ( N .x. X ) ) ) $=
+      ( cgrp wcel cz w3a wa cmul co wceq cc zcn 3ad2ant2 3ad2ant1 adantl oveq1d
+      mulcomd mulgass eqtrd ) CIJZDKJZEKJZFAJZLZMZEDNOZFBODENOZFBODEFBOBOUKULUM
+      FBUJULUMPUFUJEDUHUGEQJUIERSUGUHDQJUIDRTUCUAUBABCDEFGHUDUE $.
+  $}
+
+  ${
+    mulgmodid.b $e |- B = ( Base ` G ) $.
+    mulgmodid.o $e |- .0. = ( 0g ` G ) $.
+    mulgmodid.t $e |- .x. = ( .g ` G ) $.
+    $( Casting out multiples of the identity element leaves the group multiple
+       unchanged.  (Contributed by Paul Chapman, 17-Apr-2009.)  (Revised by AV,
+       30-Aug-2021.) $)
+    mulgmodid $p |- ( ( G e. Grp /\ ( N e. ZZ /\ M e. NN )
+                        /\ ( X e. B /\ ( M .x. X ) = .0. ) )
+                      -> ( ( N mod M ) .x. X ) = ( N .x. X ) ) $=
+      ( wcel cz wa co wceq cfv cq adantl 3ad2ant2 oveq1d cgrp w3a cmo cdiv cmul
+      cn cfl cneg cplusg cmin caddc cc0 clt wbr zq adantr nngt0 modqval syl3anc
+      nnq cc zcn nnz znq zmulcld zcnd negsubd simp1 simpl znegcld 3ad2ant3 eqid
+      flqcld mulgdir syl13anc 3eqtr2d nncn mulneg2d mulgassr oveq2 mulgz 3eqtrd
+      syl2anc eqtr3d oveq2d id mulgcl syl3an grprid ) CUAKZELKZDUFKZMZFAKZDFBNZ
+      GOZMZUBZEDUCNZFBNZEFBNZDEDUDNZUGPZUENZUHZFBNZCUIPZNZXAGXGNZXAWRWTEXDUJNZF
+      BNEXEUKNZFBNZXHWRWSXJFBWMWJWSXJOZWQWMEQKZDQKZULDUMUNZXMWKXNWLEUOUPWLXOWKD
+      UTRWLXPWKDUQREDURUSSTWRXKXJFBWMWJXKXJOWQWMEXDWKEVAKWLEVBUPWMXDWMDXCWLDLKZ
+      WKDVCRZWMXBEDVDZVMZVEVFVGSTWRWJWKXELKWNXLXHOWJWMWQVHZWMWJWKWQWKWLVIZSWRXD
+      WRDXCWMWJXQWQXRSZWMWJXCLKWQXTSVEVJWQWJWNWMWNWPVIZVKZAXGBCEXEFHJXGVLZVNVOV
+      PWRXFGXAXGWRDXCUHZUENZFBNZXFGWRYHXEFBWMWJYHXEOWQWMDXCWLDVAKWKDVQRWMXCXTVF
+      VRSTWRYIYGWOBNZYGGBNZGWRWJYGLKZXQWNYIYJOYAWRXCWRXBWMWJXBQKWQXSSVMVJZYCYEA
+      BCYGDFHJVSVOWQWJYJYKOZWMWPYNWNWOGYGBVTRVKWRWJYLYKGOYAYMABCYGGHJIWAWCWBWDW
+      EWRWJXAAKZXIXAOYAWJWJWMWKWQWNYOWJWFYBYDABCEFHJWGWHAXGCXAGHYFIWIWCWB $.
+  $}
+
+  ${
+    mulgsubdir.b $e |- B = ( Base ` G ) $.
+    mulgsubdir.t $e |- .x. = ( .g ` G ) $.
+    mulgsubdir.d $e |- .- = ( -g ` G ) $.
+    $( Subtraction of a group element from itself.  (Contributed by Mario
+       Carneiro, 13-Dec-2014.) $)
+    mulgsubdir $p |- ( ( G e. Grp /\ ( M e. ZZ /\ N e. ZZ /\ X e. B ) ) ->
```

Comment:
Comment is not correct (seems to be a copy&past error, also in set.mm). It should be "Distribution of group multiples over subtraction for group elements,  ~ subdir analog."

Reply:
Suggested wording adopted in both set.mm and iset.mm.